src/olympia/users/backends.py

```diff
@@ -21,6 +23,26 @@ def get_user(self, user_id):
             return None
 
 
+class TestUserBackend(object):
+    """Authentication backend to easily log in a user while testing."""
+    supports_anonymous_user = False
+    supports_inactive_user = False
+    supports_object_permissions = False
```

Comment:
You don't have to set those anymore since Django 1.4, all backends have to support / handle anonymous and inactive users and the `obj` parameter for permission checking. I forgot to remove them in our 1.7/1.8 upgrades.